src/wallet/walletdb.cpp

```diff
@@ -762,20 +760,22 @@ void MaybeCompactWalletDB()
         return;
     }
 
-    static unsigned int nLastSeen = CWalletDB::GetUpdateCounter();
-    static unsigned int nLastFlushed = CWalletDB::GetUpdateCounter();
+    CWalletDBWrapper& dbh = pwalletMain->GetDBHandle();
```

Comment:
In commit "CWalletDB: Store the update counter per wallet"

Maybe call this dbw for consistency with existing wallet code.